.travis.yml

```diff
@@ -88,6 +78,12 @@ cache:
 before_install:
   # Work around for Travis timeout issues, see https://github.com/travis-ci/travis-ci/issues/9112
   - sudo apt-get update --option Acquire::Retries=100 --option Acquire::http::Timeout="60"
+  # 1. Install MongoDB 3.4 and latest version of git
+  - sudo add-apt-repository -y ppa:git-core/ppa
+  - curl https://www.mongodb.org/static/pgp/server-3.4.asc | sudo apt-key add -
+  - echo "deb [arch=amd64] http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.4 multiverse" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update -qq
```

Comment:
So again, why are we doing 2 `apt-get update`s in `before_install` section?
Could this be merged into one?

Reply:
That was an oversight - I will remove the second apt-get update and move first one a couple of lines below.